userappwithbroker/src/main/java/com/microsoft/aad/adal/example/userappwithbroker/Constants.java

```diff
@@ -58,6 +59,7 @@ public String getText() {
 
 
     enum ClientId {
+        GUESTCLIENT("ea5c8087-2476-489c-ae03-ad44a2ac399d"),
         MSDEVEX("b92e0ba5-f86e-4411-8e18-6b5f928d968a"),
```

Comment:
nit: Same here. It could be better to set MSDEVEX as default.

Reply:
fixed